README.md

```diff
@@ -7,36 +7,60 @@ A frontend development for the SpaceYaTech Mastori blog application. This is a c
 
 # Space Ya Tech Blog Frontend
 
+## About
 This repository contains the frontend code for the Space Ya Tech blog application. The blog frontend is built using React and Vite, and communicates with the backend API to display and manage blog content.
 
-## Getting Started
+## Set up the project 
 
-To get started with contributing to the blog frontend, follow these steps:
+### Prerequisites 
 
-1. Clone this repository to your local machine: https://github.com/SpaceyaTech/blog-frontend.git
+To run the project on to your system, need to have the following installed .
 
-2. Install dependencies: npm install or yarn install
+- [Node](https://nodejs.org/en/download/) and [NPM packages](https://www.npmjs.com/package/download),  all latest versions.
```

Comment:
npm comes bundled in node

Reply:
hea, @nyandika so on this I should only write the latest node version as required.

Reply:
Yea, you can remove the link to npm